wrappers/python/examples/d500_triggered_calibration.py

```diff
@@ -0,0 +1,213 @@
+## License: Apache 2.0. See LICENSE file in root directory.
+## Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+import os
+import time
+import zlib
+
+import pyrealsense2 as rs
+import numpy as np
+
+'''
+aim of this script is to show how to use the triggered calibration api with safety camera,
+and to check the improvement of the depth stream before / after the running of the calibration
+'''
+
+
+def check_d500_camera_found(dev):
+    product_line = dev.get_info(rs.camera_info.product_line)
+    if product_line != "D500":
+        print("Device is not D500 Product Line - please check")
+        exit(1)
+    else:
+        print("D500 Product Line Camera found")
+
+
+def compute_fill_rate(frame):
+    frame_array = np.hstack(np.asarray(frame.get_data(), dtype=np.uint16))
+    num_of_pixels = len(frame_array)
+    fill_rate = 0
+    for pixel in frame_array:
+        if pixel > 0:
+            fill_rate += 1
+    return fill_rate / num_of_pixels * 100
+
+
+def stream_and_get_fill_rate():
+    print("Streaming frames...")
+    cfg = rs.config()
+    cfg.enable_stream(rs.stream.depth, 0, 1280, 720, rs.format.z16, 30)
+    pipe = rs.pipeline()
+    pipe.start(cfg)
+    frames_captured = 0
+    fill_rate_total = 0
+    num_of_frames = 3
+    try:
+        while frames_captured < num_of_frames:
+            frames = pipe.wait_for_frames()
+
+            depth_frame = frames.get_depth_frame()
+            fill_rate = compute_fill_rate(depth_frame)
+            fill_rate_total += fill_rate
+
+            frames_captured += 1
+
+    finally:
+        pipe.stop()
+    print("frames streamed!")
+    fill_rate_mean_value = fill_rate_total / num_of_frames
+    print("fill rate found = " + repr(fill_rate_mean_value) + "%")
+    return fill_rate_mean_value
+
+
+def build_header(new_calib_array):
+    version_major = 0
+    version_minor = 4
+    depth_table_id = 180  # 0xb4
+    depth_table_size = 496
+    counter = 0
+    test_cal_version = 0
+    crc_computed = zlib.crc32(new_calib_array)
+    version_major_ints = np.frombuffer(version_major.to_bytes(1, 'little'), dtype=np.uint8)
```

Comment:
Is this the algo version or process version? How often is this expected to change? Also for modules other than D555

Reply:
version of the depth calibration

Reply:
OK, thank you.
This means we might need to update the example code whenever algo team changes the version, even if our code does not change, but I guess it is unavoidable once they decided that the version is part of the table.